### ISSUE TITLE
Line ingredient added to list when loading fails

### ISSUE THREAD

**Issue by raacampbell** [raacampbell/lasagna#146]
Attempt to load an .csv file that's in the wrong format. Loading fails. The item is still added to the list.

**Comment by raacampbell** [comment]
Tried to reproduce this by corrupting the CSV file of a tree by removing the last number on the first line. I get this:
```
lasanga.addIngredient - Adding imagestack ingredient: result.1.mhd
lastLoadDir
recentlyLoadedFiles
Traceback (most recent call last):
  File "/home/rob/Dropbox/home/work/TissueCyte/code/lasagna/IO/tree_reader_plugin.py", line 93, in showLoadDialog
    dataTree = importData(fname,headerLine=['id','parent','z','x','y'],verbose=verbose)
  File "/home/rob/Dropbox/home/work/TissueCyte/code/lasagna/tree.py", line 79, in importData
    assert len(header)==len(dataLine)
AssertionError
```
And that leads to a failure to even start the viewer if the ```-T``` flag was used. So that needs fixing.

**Comment by raacampbell** [comment]
Could not reproduce, but I have some related fixes in: 21b534f
